Source/monster.cpp

```diff
@@ -3060,8 +3060,8 @@ void SnakeAi(int i)
 			AiDelay(monster, 15 - monster._mint + GenerateRnd(10));
 		}
 	} else {
-		if (monster._mVar1 == MM_DELAY
-		    || monster._mVar1 == MM_CHARGE
+		if (monster._mVar1 == static_cast<int>(MonsterMode::Delay)
+		    || monster._mVar1 == static_cast<int>(MonsterMode::Charge)

```

Comment:
```suggestion
		if (IsAnyOf(static_cast<MonsterMode>(monster._mVar1), MonsterMode::Delay, MonsterMode::Charge))
```

Reply:
First half is required
```suggestion
		if (static_cast<MonsterMode>(monster._mVar1) == MonsterMode::Delay
		    || static_cast<MonsterMode>(monster._mVar1) == MonsterMode::Charge
```